kubernetes/install.sh

```diff
@@ -77,17 +74,8 @@ echo ""
 
 # Download config.yaml file into temporary folder
 if [[ -z ${CONFIG_YAML_URL} ]]; then
-  save_endpoint_to_file ${BASE_CONFIG_YAML_URL} "${temp_dir}/template_config.yaml"
-
-  # Iterate over TEMPLATE_NAMES and download their config.yaml files
-  for template in ${TEMPLATE_NAME}
-  do
-      echo "Downloading config.tmpl file for template '${template}'..."
-      CONFIG_YAML_URL="${REPO_BASE_URL}/kubernetes/${template}_config.tmpl"
-      save_endpoint_to_file ${CONFIG_YAML_URL} "${temp_dir}/${template}_config.tmpl"
-      cat ${temp_dir}/${template}_config.tmpl >> ${temp_dir}/template_config.yaml
-      echo "Added ${template}."
-  done
+  echo "No custom config.yaml file configuration found. Please spciy a custom config.yaml file using the 'CONFIG_YAML_URL' variable."
```

Comment:
Cant we add support for no configuration as well?
The exporter should support empty configuration, and allow it to be handled via UI only.

I think for this you will also need to update the k8s-exporter chart, and place a default `""` value in the the `config.configMap` value.